adapters/cwire/cwire.go

```diff
@@ -0,0 +1,111 @@
+package cwire
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+/*
+Your bid adapter code will need to implement and export:
+
+- The adapters.Builder method to create a new instance of the adapter based on
+  the host configuration
+
+- The adapters.Bidder interface consisting of the MakeRequests method to create
+  outgoing requests to your bidding server and the MakeBids method to create bid
+  responses.
+*/
+
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the CWire adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+/*
+The MakeRequests method is responsible for returning none, one, or many HTTP
```

Comment:
This comment seems to be a copy paste of the interface definition of this function. Can you please modify it to just provide a description of the CWire implementation of `MakeRequests`? Similar comment for `MakeBids`

Reply:
Done